### PR TITLE
avoid schema change during the upgrade and workaround list order issue

### PR DESCRIPTION
**Pull request by amoskong**
## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I gave variables/functions meaningful self-explanatory names
- [x] I didn't leave commented-out/debugging code
- [x] I didn't copy-paste code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by amoskong** [comment]
Tested this PR by https://jenkins.scylladb.com/job/scylla-staging/job/amos/job/rolling-upgrade-centos7-test/11

The job failed for scylla issue. scylla issue of schema change and list order / list item repeat issue didn't occur. So it works as expected.

**Comment by amoskong** [comment]
> You added a workaround for list_test but you didn’t take it out of the commented block.
> Is it on purpose?

Yes, it's expected.

The delay might help to avoid the list order issue, but I still want to keep disabling the test in official jobs until it's fixed.

**Comment by amoskong** [comment]
> > You added a workaround for list_test but you didn’t take it out of the commented block.
> > Is it on purpose?
> 
> Yes, it's expected.
> 
> The delay might help to avoid the list order issue, but I still want to keep disabling the test in official jobs until it's fixed.

Just bring the list_test back as suggested by Roy.

File: sdcm/fill_db_data.py [hunk 1]
```diff
@@ -2972,20 +2996,10 @@ def fill_db_data(self):
...
-
     def verify_db_data(self):
         # Prepare connection
         node = self.db_cluster.nodes[0]
         with self.cql_connection_patient(node, keyspace='keyspace_fill_db_data') as session:
             # override driver consistency level
             session.default_consistency_level = ConsistencyLevel.QUORUM
             self.run_db_queries(session, session.default_fetch_size)
-
-    def clean_db_data(self):
```
Comment (bentsi) [inline sdcm/fill_db_data.py]: why did you remove it?

Comment (amoskong) [reply sdcm/fill_db_data.py]: It's never used.
This PR replaced the clean_db_data() by truncate
